app/validators/metadata_validator.py

```diff
@@ -29,11 +30,19 @@ def validate_duplicates(self):
 
     def validate_mandatory(self):
         # user_id and period_id required downstream for receipting
-        # ru_name required for template rendering in default and NI theme
-        required_metadata_names = ["user_id", "period_id"]
-
-        if self.theme_name in ["default", "northernireland"]:
-            required_metadata_names.append("ru_name")
+        # ru_name required for template rendering in business, default and NI theme
+        # display_address required for template rendering in census, census-nisra theme when form_type exists
+        required_metadata_names = []
+
+        if self.theme_name in ["business", "default", "northernireland"]:
+            required_metadata_names.extend(["user_id", "period_id", "ru_name"])
```

Comment:
Once RAS moves to v2 format, we can remove `user_id` and `period_id`.

Reply:
Ok, I will keep it for now!